CMake/README.md

```diff
@@ -43,6 +43,16 @@
 
       set(CMAKE_CXX_FLAGS "-std=c++11 -stdlib=libc++ ${CMAKE_CXX_FLAGS}")
 
+      if (MSVC)
+        # /wd4005  macro-redefinition
+        # /wd4068  unknown pragma
+        # /wd4244  conversion from 'type1' to 'type2'
+        # /wd4267  conversion from 'size_t' to 'type2'
+        # /wd4800  force value to bool 'true' or 'false' (performance warning)
+        add_compile_options(/wd4005 /wd4068 /wd4244 /wd4267 /wd4800)
```

Comment:
Consider these warning flags as temporary fix just to get Abseil to build on Windows. We should eventually fix these warnings or at least reduce the number of occurrence until we can just use #pragma explicitly.

Reply:
On second thought, I think trying to silence MSVC about these warnings by modifying the code might be impossible. MSVC is more noisy than GCC and Clang. Some projects have even longer list (see LLVM and Chromium for instance).